index.html

```diff
@@ -2086,15 +2086,20 @@ <h3 class="heading settled" data-level="3.2" id="is-origin-trustworthy"><span cl
   return "<code>Potentially Trustworthy</code>".</p>
       <p class="note" role="note"><span>Note:</span> This is meant to be analog to the <a data-link-type="dfn" href="https://w3c.github.io/webappsec-mixed-content/#a-priori-authenticated-url" id="ref-for-a-priori-authenticated-url"><i lang="la">a priori</i> authenticated URL</a> concept in <a data-link-type="biblio" href="#biblio-mix">[MIX]</a>.</p>
      <li data-md>
-      <p>If <var>origin</var>’s <a data-link-type="dfn" href="https://html.spec.whatwg.org/multipage/origin.html#concept-origin-host" id="ref-for-concept-origin-host">host</a> component matches one of the CIDR
+      <p>If <var>origin</var>’s <a data-link-type="dfn" href="https://html.spec.whatwg.org/multipage/origin.html#concept-origin-host" id="ref-for-concept-origin-host">host</a> matches one of the CIDR
   notations <code>127.0.0.0/8</code> or <code>::1/128</code> <a data-link-type="biblio" href="#biblio-rfc4632">[RFC4632]</a>, return "<code>Potentially Trustworthy</code>".</p>
      <li data-md>
-      <p>If <var>origin</var>’s <a data-link-type="dfn" href="https://html.spec.whatwg.org/multipage/origin.html#concept-origin-host" id="ref-for-concept-origin-host①">host</a> component is "<code>localhost</code>" or
-  falls within "<code>.localhost</code>", and the user agent conforms to the name
-  resolution rules in <a data-link-type="biblio" href="#biblio-let-localhost-be-localhost">[let-localhost-be-localhost]</a>, return "<code>Potentially Trustworthy</code>".</p>
+      <p>If the user agent conforms to the name resolution rules in <a data-link-type="biblio" href="#biblio-let-localhost-be-localhost">[let-localhost-be-localhost]</a> and one of the following is true:</p>
+      <ul>
+       <li data-md>
+        <p><var>origin</var>’s <a data-link-type="dfn" href="https://html.spec.whatwg.org/multipage/origin.html#concept-origin-host" id="ref-for-concept-origin-host①">host</a> is "<code>localhost</code>" or "<code>localhost.</code>"</p>
+       <li data-md>
+        <p><var>origin</var>’s <a data-link-type="dfn" href="https://html.spec.whatwg.org/multipage/origin.html#concept-origin-host" id="ref-for-concept-origin-host②">host</a> ends with "<code>.localhost</code>" or "<code>.localhost</code>"</p>
```

Comment:
It looks like the compiler is removing the trailing period from the second code block here? It is supposed to be ```"<code>.localhost.</code>"```, according to the source.

(Or maybe index.html is just out of date w.r.t index.src.html?)

Reply:
Thanks, fixed.